pkg/optimization/details.go

```diff
@@ -288,7 +281,7 @@ func (stats *SubstatOptimizerDetails) optimizeERSubstats(tolMean float64, tolSD
 	)
 
 	for idxChar, char := range stats.charProfilesERBaseline {
-		charDebug = stats.findOptimalERforChar(idxChar, char, tolMean, tolSD)
+		stats.findOptimalERforChar(idxChar, char, tolMean, tolSD)
 		opDebug = append(opDebug, charDebug...)
```

Comment:
nit picking here but the `charDebug` variable should just be deleted altogether since it doesn't seem to be used

Reply:
Not a nitpick, we should remove these whenever it's apparent.